benchexec/tools/aise.py

```diff
@@ -0,0 +1,67 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.result as result
+import benchexec.tools.template
+import benchexec.model
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for AISE(https://github.com/ZhenWang233/AISE).
+    """
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("aise", subdir="bin")
+
+    def cmdline(self, executable, options, task, rlimits):
+        return [executable] + options + [task.single_input_file]
```

Comment:
Just noticed: If you want to participate in SV-COMP'24, are you sure you do not need additional information from the task, in particular the data model?

Reply:
Thank you for your suggestion, but I want to keep it in the current version. I used a hard-code to built-in a property (unreachcall, ILP32) in our tool. Because we only focus on a subcategory"ReachSafety-Loops" in SV-COMP'24, their data models are same, so built-in a property can work in most cases. I known it is not a good handling, but changing it need some time and testing.